.github/workflows/ci.yml

```diff
@@ -89,6 +89,40 @@ jobs:
         name: docs
         path: site
 
+# TODO: Get this working once the memory leak issue is resolved - https://github.com/pydantic/pydantic/issues/5136
```

Comment:
I can confirm the memory leak issue is solved and the memray test is passing for me locally; not sure exactly how this should be modified for compatibility with v2.